src/server/server_family.h

```diff
@@ -297,6 +300,10 @@ class ServerFamily {
   std::unique_ptr<util::fb2::FiberQueueThreadPool> fq_threadpool_;
   std::shared_ptr<detail::SnapshotStorage> snapshot_storage_;
 
+  Mutex bg_save_mu_;
+  util::fb2::Fiber bg_save_fb_;
+  bool bg_save_fb_done_ = false;
```

Comment:
I would have loved a polling interface on a fiber, anyway I can move this to separate class for now. ServerFamilly members are exploding :rofl: